tracer/src/Datadog.Trace/Debugger/ExceptionAutoInstrumentation/ExceptionCaseInstrumentationManager.cs

```diff
@@ -115,7 +115,14 @@ private static List<MethodUniqueIdentifier> GetMethodsToRejit(ParticipatingFrame
 
         internal static void Revert(ExceptionCase @case)
         {
-            Log.Information("Reverting {ExceptionCase}", @case);
+            try
+            {
+                Log.Information("Reverting {ExceptionCase}", @case);
+            }
+            catch (Exception ex)
+            {
+                Log.Error(ex, "Failed to log an exception case while reverting...");
```

Comment:
I don't like the look of this pattern 😬 _Why_ would you not be able to log an `ExceptionCase`?

Reply:
I have seen NRE bubbling out from surrounding code (via instrumentation telemetry) and couldn't figure out why, so I've added it just in case. `ExceptionCase` overrides `ToString` and has several fields that too override it etc.. couldn't put my hand on NRE from looking at the code (plus everything is annotated with `#nullable enable`)...